conf/cndp_upf_4worker.jsonc

```diff
@@ -0,0 +1,289 @@
+    //    description   - (O) The description, 'desc' can be used as well
+    "lports": {
+        "enp134s0:0": {
+            "pmd": "net_af_xdp",
+            "qid": 22,
+            "umem": "umem0",
+            "region": 0,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Access LAN 0 port"
+        },
+        "enp134s0:1": {
+            "pmd": "net_af_xdp",
+            "qid": 23,
+            "umem": "umem1",
+            "region": 0,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Access LAN 1 port"
+        },
+        "enp134s0:2": {
+            "pmd": "net_af_xdp",
+            "qid": 24,
+            "umem": "umem2",
+            "region": 0,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Access LAN 2 port"
+        },
+        "enp134s0:3": {
+            "pmd": "net_af_xdp",
+            "qid": 25,
+            "umem": "umem3",
+            "region": 0,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Access LAN 3 port"
+        },
+        "enp136s0:0": {
+            "pmd": "net_af_xdp",
+            "qid": 22,
+            "umem": "umem0",
+            "region": 1,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Core LAN 0 port"
+        },
+        "enp136s0:1": {
+            "pmd": "net_af_xdp",
+            "qid": 23,
+            "umem": "umem1",
+            "region": 1,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Core LAN 1 port"
+        },
+        "enp136s0:2": {
+            "pmd": "net_af_xdp",
+            "qid": 24,
+            "umem": "umem2",
+            "region": 1,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Core LAN 2 port"
+        },
+        "enp136s0:3": {
+            "pmd": "net_af_xdp",
+            "qid": 25,
+            "umem": "umem3",
+            "region": 1,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Core LAN 3 port"
+        }
+    },
+
+    // (O) Define the lcore groups for each thread to run
+    //     Can be integers or a string for a range of lcores
+    //     e.g., [10], [10-14,16], [10-12, 14-15, 17-18, 20]
+    // Names of a lcore group and its lcores assigned to the group.
+    // The initial group is for the main thread of the application.
+    // The default group is special and is used if a thread if not assigned to a group.
+    //
+    // CNDP BESS Port does not use this section.
```

Comment:
Is this comment in the correct place? I am asking this because I see that in the 1worker and 8worker files, it is in the `options` below